File: FrontEnd(Angular 9)/src/app/_shared/model/employer.ts
export class employer {

    id:number;
    nom:string;
    prenom:string;
    tel:string;
    type:string;
    superieur:number;
    login:string;
    mdp:string;


    getId():number{
        return this.id;
    }
    getNom():string{
        return this.nom;
    }
    getPrenom():string{
        return this.prenom;
    }
    getType():string{
        return this.type;
    }
    getSuperieur():number{
        return this.superieur;
    }
    getLogin():string{
        return this.login;
    }
    getTel():string{
        return this.tel;
    }
    getMdp():string{
        return this.mdp;
    }
   
    setId(id:number){
        this.id=id;
    }
    setNom(nom:string){
        this.nom=nom;
    }
    setPrenom(prenom:string){
        this.prenom=prenom;
    }
    setSuperieur(sup:number){
        this.superieur=sup;
    }
    setLogin(login:string){
        this.login=login;
    }
    setTel(tel:string){
        this.tel=tel;
    }
    setMdp(mdp:string){
        this.mdp=mdp;
    }
    setType(type:string){
        this.type=type;
    }
    
}

File: BackEnd(Java -jersy)/CompteBancaireWEBAPI/src/model/employer.java
package model;

public class employer {
	private int id;
	private String nom,prenom,tel,type;
	private String mail;
	private int superieur;
	private String login;
	private String mdp;
	
	
	public employer() {
	}




	public String getMail() {
		return mail;
	}




	public void setMail(String mail) {
		this.mail = mail;
	}




	public int getId() {
		return id;
	}


	public void setId(int id) {
		this.id = id;
	}


	public String getNom() {
		return nom;
	}


	public void setNom(String nom) {
		this.nom = nom;
	}


	public String getPrenom() {
		return prenom;
	}


	public void setPrenom(String prenom) {
		this.prenom = prenom;
	}


	public String getTel() {
		return tel;
	}


	public void setTel(String tel) {
		this.tel = tel;
	}


	public String getLogin() {
		return login;
	}


	public void setLogin(String login) {
		this.login = login;
	}


	public String getMdp() {
		return mdp;
	}


	public void setMdp(String mdp) {
		this.mdp = mdp;
	}


	public String getType() {
		return type;
	}


	public void setType(String type) {
		this.type = type;
	}


	public int getSuperieur() {
		return superieur;
	}


	public void setSuperieur(int superieur) {
		this.superieur = superieur;
	}
	
	

}
